pallets/domains/src/lib.rs

```diff
@@ -411,19 +417,20 @@ pub mod pallet {
                 Error::<T>::TooManyDomainsPerAccount,
             );
 
-            let mut deposit = Zero::zero();
-            if let IsForced::No = is_forced {
-                // TODO: unreserve the balance for expired or sold domains
-                deposit = T::BaseDomainDeposit::get();
-                <T as Config>::Currency::reserve(&owner, deposit)?;
-            }
+            let deposit_info: DomainDepositOf<T> = match caller {
+                CallerAccount::Account(acc) => (acc, T::BaseDomainDeposit::get()),
+                CallerAccount::ForceOrigin => (owner.clone(), Zero::zero()),
+            }.into();
+
+            // TODO: unreserve the balance for expired or sold domains
+            <T as Config>::Currency::reserve(&deposit_info.depositor, deposit_info.deposit)?;
```

Comment:
Do we have `unreserve()` for this deposit somewhere?

Reply:
We don't, because there are no any use cases in the pallet yet, when we need to unreserve token.